### PR TITLE
Initial implementation for including additional pre-release information

### LINKED ISSUE #12: Include information about pre-release versions
When viewing details about an update, WordPress shows the readme. 

For pre-release versions, let's prepend some extra information.

1. Some warning text that this is a pre-release.
2. A link through to the github tag.
3. Pull and display data about a beta/release from GitHub API. We can include information about features and what to focus on testing in the release on GitHub (release description).

### PR DESCRIPTION
**Pull request by bor0**
Resolves #12.

### CODE REVIEW

**Comment by bor0** [comment]
untitled12345

File: includes/class-wc-beta-tester.php [hunk 1]
```diff
@@ -344,6 +355,40 @@ public function upgrader_source_selection( $source, $remote_source, $upgrader )
...
+			return $res;
+		}
+
+		// Not a pre-release, no need to do anything.
+		if ( ! $this->is_prerelease( $res->version ) ) {
+			return $res;
+		}
+
+		if ( isset( $res->sections['description'] ) ) {
+			$res->sections['description'] = __( '<h1><span>&#9888;</span>This is a pre-release<span>&#9888;</span></h1>', 'woocommerce-beta-tester' )
```
Comment (mikejolley) [inline includes/class-wc-beta-tester.php:378]: This is good for now. We can include more info later.

File: includes/class-wc-beta-tester.php [hunk 2]
```diff
@@ -344,6 +355,40 @@ public function upgrader_source_selection( $source, $remote_source, $upgrader )
...
+				. $res->sections['description'];
+		}
+
+		$res->sections['pre-release_information'] = $this->get_version_information( $res->version );
```
Comment (mikejolley) [inline includes/class-wc-beta-tester.php]: Should we run some formatting over this? I.e. autop and make_clickable?

**Review by mikejolley — APPROVED** [review]
Small comment but other than that, good to go.

**Pull request merged**